addons/test_mail/models/test_mail_models.py

```diff
@@ -40,6 +40,7 @@ class MailTestStandard(models.Model):
     user_id = fields.Many2one('res.users', 'Responsible', tracking=True)
     umbrella_id = fields.Many2one('mail.test', tracking=True)
     company_id = fields.Many2one('res.company')
+    state = fields.Selection([('ready', 'Ready'), ('done', 'Done')], string="State", default='ready')
```

Comment:
Not sure we can do it in stable cc @Xavier-Do even if it is a test module, it is still adding a field in stable ^^ .

Reply:
If we can't do it I can remove the field and keep the 'default_state' in the context within my test method.
It makes a bit less sense but it's still a relevant test.

What do you think?

Reply:
Probably better, with a note telling 'to be improved in master', and in master we add the field to have a complete test.

Reply:
Test modules are not installed in upgrade databases anymore to avoid blocking changes in stable, so you can do what you want!

Sorry tde, I won't be your excuse to keep the test mail models are as they are! ^^

Reply:
Yes, but if someone in the world has this module installed ... not sure it is worth it. Test without field already prevent a regression.